fastlane/Fastfile

```diff
@@ -839,3 +892,26 @@ platform :ios do
     UI.message("Gutenberg version: #{(res.scan(/'([^']*)'/))[0][0]}")
   end
 end
+
+def read_version_from_config
```

Comment:
💡  If you find it useful, it could be nice to add a fixme comment until you tackle this in separate PR (except if you plan to do it like super soon)
```suggestion
# FIXME: Move to release-toolkit
def read_version_from_config
```